### DISCUSSION TITLE
[Q] Aero glass

### DISCUSSION THREAD

**Discussion by shikulja** [valinet/ExplorerPatcher#239] [General]
There are no more utilities for restoring aero glass, maybe there is a way to get it back?
In opensource, this may be the most popular solution.

**Comment by valinet** [comment]
From my reply on a Reddit thread discussing things like BM’s Aero Glass 8:

>> The architecture of DWM changed quite a while since OS version 2004. Quite a lot of work would have been required to make it work on newer versions, and the developer never went on with the project. Unfortunately, he did not publish the source either, so most of the knowledge is lost, or rather, lost behind a supplementary wall represented by the need to decipher and make sense of its disassembly. With this also being a very niche feature, there is little incentive for someone to pick off where he/she left off.

It doesn’t matter if it is open source or not, someone still has to take the time and investigate the matter etc. Simply being open source does not mean it writes itself automatically or make it any easier.

Thanks
